src/Interface/Modules/Render/ES/SRInterface.cc

```diff
@@ -796,15 +796,15 @@ glm::vec2 ScreenParams::positionFromClick(int x, int y) const
     //----------------------------------------------------------------------------------------------
     void SRInterface::updateClippingPlanes()
     {
-      StaticClippingPlanes* clippingPlanes = mCore.getStaticComponent<StaticClippingPlanes>();
-      if (clippingPlanes)
+      auto* clippingPlanes = mCore.getStaticComponent<StaticClippingPlanes>();
+      if (clippingPlanes && sceneBBox_.valid())
```

Comment:
This line fixes the error when enabling a clipping plane with an empty viewscene